charts/jaeger/templates/es-index-cleaner-cronjob.yaml

```
@@ -80,4 +80,10 @@ spec:
           tolerations:
             {{- toYaml . | nindent 12 }}
         {{- end }}
+          volumes: 
+          {{- range .Values.esIndexCleaner.extraSecretMounts }}
+          - name: {{ .name }}
+            secret:
+              secretName: {{ .secretName }}
```

```suggestion
          - name: {{ .name }}
            secret:
              secretName: {{ .secretName }}
```

nit: Would you mind indenting these? other lists are indented in the chart.

---

Yes, sure! Done.